src/client/render/cameras.js

```diff
@@ -80,21 +101,32 @@ export function CameraFlyControls(
 		// Hook pointer lock state change events
 		func = function() {
 			// document.pointerLockElement is null if pointerlock is inactive
-			if (document.pointerLockElement !== options.pointerLockElement) stop();
+			if (document.pointerLockElement !== options.pointerLockElement) self.stop();
 		};
-		listeners.push( addRegisteredEventListener(document, "pointerlockchange", func, false) );
+		_listeners.push( addRegisteredEventListener(document, "pointerlockchange", func, false) );
 
 		// Request pointerlock
 		func = function() {
-			controls.lock();
+			self.controls.lock();
 		};
-		listeners.push( addRegisteredEventListener(options.pointerLockElement, "mousedown", func, false) );
+		_listeners.push( addRegisteredEventListener(options.pointerLockElement, "mousedown", func, false) );
 
 		// Release pointerlock
 		func = function() {
-			controls.unlock();
+			self.controls.unlock();
 		};
-		listeners.push( addRegisteredEventListener(document, "mouseup", func, false) );
+		_listeners.push( addRegisteredEventListener(document, "mouseup", func, false) );
```

Comment:
Some of these event listeners could be a bit cleaner. This also works, without needing to set up `func` or using `self`:
`_listeners.push( addRegisteredEventListener(document, "mouseup", () => {this.controls.unlock();}, false) );`
I guess it's personal preference in the end though.

Reply:
Not going to bother. I kinda like `func` here because it allows the `push` to `_listeners` to be on a single line. Good suggestion though (: